### ISSUE TITLE
Readonly, ReadonlyTable controls should be accessible

### ISSUE THREAD

**Issue by nmgokhale** [elyra-ai/canvas#484]
Fix a11y violations in these controls - 

**Readonly control -**
<img width="323" alt="Screen Shot 2021-02-16 at 9 59 30 AM" src="https://user-images.githubusercontent.com/25124000/108102726-c546ce80-703d-11eb-87f4-803312d09b2e.png">

**ReadonlyTable control -**
Remove **"subpanel"** column in readonlyTable because its always an empty column.
![Screen Shot 2021-02-15 at 12 33 47 PM](https://user-images.githubusercontent.com/25124000/108102844-f58e6d00-703d-11eb-9b41-f557b8aeb2b3.png)

**Add aria-labels for title editor and resize buttons -** 

![Screen Shot 2021-02-15 at 9 24 26 AM](https://user-images.githubusercontent.com/25124000/108103022-2f5f7380-703e-11eb-9799-4425f316570d.png)
